proto/parca/query/v1alpha1/query.proto

```diff
@@ -331,6 +334,39 @@ message FlamegraphNodeMeta {
   parca.metastore.v1alpha1.Line line = 4;
 }
 
+// CallgraphNode represents a node in the graph
+message CallgraphNode {
+  // id is the unique id of the node
+  string id = 1;
+
+  // name is the name of the node
+  string name = 2;
+}
+
+// CallgraphEdge represents an edge in the graph
+message CallgraphEdge {
+  // id is the unique id of the edge
+  string id = 1;
+
+  // source represents the id of the source node
+  string source = 2;
+
+  // target represents the id of the target node
+  string target = 3;
+
+  // visits is the number of visits to the edge
+  int64 visits = 4;
```

Comment:
it's not about the number of visits, what we want to understand here is the amount of cumulative that this edge leads to